Tensile/Tests/pre_checkin/direct_to_vgpr/dtv_sgemm_lite.yaml

```diff
@@ -42,7 +42,8 @@ BenchmarkProblems:
         #  - [ 8, 8, 1 ]
         - DepthU: [16,32]
         - ExpandPointerSwap: [1] # 1 for DirectToVgpr
-        - LdsBlockSizePerPad: [-1]
+        - LdsBlockSizePerPadA: [-1]
+        - LdsBlockSizePerPadB: [-1]
```

Comment:
Can also just remove since new default is -1. Most other tests would have used a default of 0, do we need a test with a specific value of 0 now that default has changed?

Reply:
We can remove default setting.
Also, I can change some tests with LBSPP=0.

Reply:
Updated. I updated some tests to use LBSPP=0.